shellborn/src/render/svg/render_able.rs

```diff
@@ -11,26 +12,30 @@ pub trait SvgRenderAble {
 
 impl SvgRenderAble for Positioned<Entity> {
     fn render_onto(&self, document: Document) -> Document {
-        let width = 100;
-        let height = 50;
+        let width = self.width();
+        let height = self.height();
 
-        let ellipse = Ellipse::new()
-            .set("cx", self.center_x as i32)
-            .set("cy", self.center_y as i32)
-            .set("rx", width / 2)
-            .set("ry", height / 2);
+        let ellipse = Rectangle::new()
```

Comment:
```suggestion
        let body = Rectangle::new()
```
Or something like that but not ellipse

Reply:
done